### PR TITLE
Added guide for setting redirect config for journal on staging

### PR DESCRIPTION
**Pull request by erezmus**
Part of elifesciences/issues#5222

### CODE REVIEW

File: docs/deployment.md [hunk 1]
```diff
@@ -33,6 +33,33 @@ Consult the documentation on each service for more information on how they can b
 
 * Audit (@libero/audit)
 
+### Journal login
+
+TODO: move this somewhere else
+
+For journal authentication, a configuration setting needs to be set to ensure the redirect returns the user to the correct url after authentication. The following
+describe how to change this setting in each environment.
```
Comment (giorgiosironi) [inline docs/deployment.md]: state that `builder-configuration` changes require admin approval (means @diversemix)

File: docs/deployment.md [hunk 2]
```diff
@@ -33,6 +33,33 @@ Consult the documentation on each service for more information on how they can b
...
+(this needs credentials, ask @erezmus)
+
+1. : Change https://github.com/elifesciences/builder-configuration/blob/master/pillar/environment-continuumtestpreview-public.sls#L24
+1. : Change https://github.com/elifesciences/elife-xpub/blob/develop/config/staging.js#L14-L19
+1. : Run https://alfred.elifesciences.org/job/test-journal/
+
+#### Prod (journal environment: prod)
+URL: https://elifesciences.org
+
+1. : Change https://github.com/elifesciences/builder-configuration/blob/master/pillar/environment-prod-public.sls#L75
```
Comment (diversemix) [inline docs/deployment.md]: ... to what value?

File: docs/deployment.md [hunk 3]
```diff
@@ -33,6 +33,33 @@ Consult the documentation on each service for more information on how they can b
...
+
+TODO: move this somewhere else
+
+For journal authentication, a configuration setting needs to be set to ensure the redirect returns the user to the correct url after authentication. The following
+describe how to change this setting in each environment. Note that builder-configuration changes requires admin approval (@diversimix)
+
+#### Staging (journal environment: continuumtest):
+URL: https://continuumtest--cdn-journal.elifesciences.org/
+
+1. : Change https://github.com/elifesciences/builder-configuration/blob/master/pillar/environment-continuumtest-public.sls#L58
```
Comment (diversemix) [inline docs/deployment.md]: ...to what value?

File: docs/deployment.md (hunk 2 above)
Comment (giorgiosironi) [reply docs/deployment.md]: In all of these cases it should be the `/login` URL of Xpub or Reviewer

File: docs/deployment.md (hunk 3 above)
Comment (diversemix) [reply docs/deployment.md]: We should also mention the review process here ... as not everyone can approve changes in builder.

Comment (erezmus) [reply docs/deployment.md]: i think it does say that...

Comment (giorgiosironi) [reply docs/deployment.md]: https://github.com/libero/reviewer/pull/500#discussion_r350268461

File: docs/deployment.md [hunk 4]
```diff
@@ -33,6 +33,33 @@ Consult the documentation on each service for more information on how they can b
 
 * Audit (@libero/audit)
 
+### Journal login
+
+TODO: move this somewhere else
+
+For journal authentication, a configuration setting needs to be set to ensure the redirect returns the user to the correct url after authentication. The following
+describe how to change this setting in each environment. Note that builder-configuration changes requires admin approval (@diversimix)
```
Comment (giorgiosironi) [inline docs/deployment.md]: wrong tag

File: docs/deployment.md [hunk 5]
```diff
@@ -33,6 +33,33 @@ Consult the documentation on each service for more information on how they can b
...
+1. : Change https://github.com/elifesciences/builder-configuration/blob/master/pillar/environment-continuumtest-public.sls#L58
+1. : Run https://alfred.elifesciences.org/job/test-journal/
+
+#### XPub staging (journal environment: continuumtestpreview)
+URL: https://continuumtestpreview--journal.elifesciences.org/
+(this needs credentials, ask @erezmus)
+
+1. : Change https://github.com/elifesciences/builder-configuration/blob/master/pillar/environment-continuumtestpreview-public.sls#L24
+1. : Change https://github.com/elifesciences/elife-xpub/blob/develop/config/staging.js#L14-L19
+1. : Run https://alfred.elifesciences.org/job/test-journal/
```
Comment (giorgiosironi) [inline docs/deployment.md]: missing: `1. : Wait for https://alfred.elifesciences.org/job/test-elife-xpub-source-repository and https://alfred.elifesciences.org/job/test-elife-xpub

**Pull request merged**